packages/appium/lib/extension/manifest-migrations.js

```diff
@@ -26,7 +26,7 @@ const Migrations = {
    */
   [SCHEMA_REV_3]: (data) => {
     let shouldSync = false;
-    const allExtData = [...Object.values(data.drivers), ...Object.values(data.plugins)];
+    const allExtData = [...Object.values(data.drivers || {}), ...Object.values(data.plugins || {})];
```

Comment:
while these shouldn't ever be be `null` or `undefined` in practice, I guess it's possible.